src/rdb.c

```diff
@@ -1278,11 +1278,7 @@ int rdbSaveRio(rio *rdb, int *error, int rdbflags, rdbSaveInfo *rsi) {
         di = NULL; /* So that we don't release it again on error. */
     }
 
-    /* If we are storing the replication information on disk, persist
-     * the script cache as well: on successful PSYNC after a restart, we need
-     * to be able to process any EVALSHA inside the replication backlog the
-     * master will send us. */
-    if (rsi && dictSize(server.lua_scripts)) {
+    if (dictSize(server.lua_scripts)) {
```

Comment:
I don't it is a good idea to remove comments

Reply:
I think this comment is not consistent with the code, because now the lua script is always saved in rdb, so I deleted it.